requirements.txt

```diff
@@ -12,3 +12,4 @@ adafruit-circuitpython-simpleio
 adafruit-circuitpython-minimqtt
 adafruit-circuitpython-dps310
 adafruit-circuitpython-ahtx0
+sphinx>=4.0.0
```

Comment:
This needs to go into a new `/docs/requirements.txt` file, not in the `/requirements.txt` file. Please create the new file, add this to it, and remove it from here.

Reply:
@tekktrik I just realised you did it on all your PRs. I didn't notice it until now. Please go back through and submit another PR to fix this issue. Apologies for not being clearer about it in the beginning.

Reply:
Yeah, thank you for catching that, I'll go make those changes now

Reply:
@tekktrik I think this line still needs to be removed - unless the diff is acting up.